nucleus/metrics/filters.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Tuple
+
+from nucleus.annotation import Annotation
+from nucleus.prediction import Prediction
+
+from .base import AnnotationOrPredictionList, Filter, PredictionFilter
+from .polygon_utils import BoxOrPolygonAnnotation, _polygon_annotation_to_shape
```

Comment:
I'd convert the second method to public if you are using it outside of the module.